sql-plugin/src/main/scala/org/apache/spark/sql/rapids/execution/GpuHashJoin.scala

```diff
@@ -928,16 +928,13 @@ trait GpuHashJoin extends GpuExec {
   def doJoin(
       builtBatch: ColumnarBatch,
       stream: Iterator[ColumnarBatch],
-      targetSize: Long,
+      realTarget: Long,
```

Comment:
nit: This is a bit confusing why is it named realTarget. Before there was context now there is not. Could you at least add in a comment?

Reply:
Restored the name.